playground/mocks/data/idp/idx/authenticator-verification-custom-app-push.json

```diff
@@ -0,0 +1,244 @@
+        ],
+        "href":"http://localhost:3000/idp/idx/challenge/poll",
+        "method":"POST",
+        "produces":"application/ion+json; okta-version=1.0.0",
+        "refresh":4000,
+        "value":[
+          {
+            "name":"stateHandle",
+            "required":true,
+            "value":"02XxswQHZqCz4aWbqQFNXYRC4HHoqd76NBC-FIGbbH",
+            "visible":false,
+            "mutable":false
+          }
+        ],
+        "accepts":"application/json; okta-version=1.0.0"
+      },
+      {
+        "rel":[
+          "create-form"
+        ],
+        "name":"select-authenticator-authenticate",
+        "href":"http://localhost:3000/idp/idx/challenge",
+        "method":"POST",
+        "produces":"application/ion+json; okta-version=1.0.0",
+        "value":[
+          {
+            "name":"authenticator",
+            "type":"object",
+            "options":[
+              {
+                "label":"Custom Push",
+                "value":{
+                  "form":{
+                    "value":[
+                      {
+                        "name":"id",
+                        "required":true,
+                        "value":"aut198w4v0f8dr8gT0g4",
+                        "mutable":false
+                      },
+                      {
+                        "name":"methodType",
+                        "type":"string",
+                        "required":false,
+                        "options":[
+                          {
+                            "label":"Get a push notification",
+                            "value":"push"
+                          }
+                        ]
+                      }
+                    ]
+                  }
+                },
+                "relatesTo":"$.authenticators.value[1]"
+              },
+              {
+                "label":"Duo Security",
+                "value":{
+                  "form":{
+                    "value":[
+                      {
+                        "name":"id",
+                        "required":true,
+                        "value":"aut1bnark0PdN44oq0g4",
+                        "mutable":false
+                      },
+                      {
+                        "name":"methodType",
+                        "required":false,
+                        "value":"duo",
+                        "mutable":false
+                      }
+                    ]
+                  }
+                },
+                "relatesTo":"$.authenticatorEnrollments.value[0]"
+              }
+            ]
+          },
+          {
+            "name":"stateHandle",
+            "required":true,
+            "value":"02XxswQHZqCz4aWbqQFNXYRC4HHoqd76NBC-FIGbbH",
+            "visible":false,
+            "mutable":false
+          }
+        ],
+        "accepts":"application/json; okta-version=1.0.0"
+      }
+    ]
+  },
+  "currentAuthenticator":{
+    "type":"object",
+    "value":{
+      "resend":{
```

Comment:
this should be a `challenge-poll` right? Which is what the remediation form above contains

Reply:
Actually this shouldn't be there at all. The resend is a completely diff api response. I cross checked with OV as well. Copied over form the wiki will update that too.